### PR TITLE
docs: fix grammatical error on text-overflow page

### PR DESCRIPTION
**Pull request by idorenyinudoh**
### Description
fix grammatical error on text-overflow page

### CODE REVIEW

**Comment by teoli2003** [comment]
Hi @idorenyinudoh ! Thanks for your proposal but the original is correct grammar while your change is incorrect. Therefore I'm closing this PR.

**Comment by idorenyinudoh** [comment]
it's misleading man
misleading
there should be a comma somewhere at least

**Comment by teoli2003** [comment]
Oh yes, we can add a comma.

File: files/en-us/web/css/text-overflow/index.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The **`text-overflow`** [CSS](/en-US/docs/Web/CSS) property sets how hidden over
 
 {{EmbedInteractiveExample("pages/css/text-overflow.html")}}
 
-The `text-overflow` property doesn't force an overflow to occur. To make text overflow its container you have to set other CSS properties: {{cssxref("overflow")}} and {{cssxref("white-space")}}. For example:
+The `text-overflow` property doesn't force an overflow to occur. To make text overflow, on its container you have to set other CSS properties: {{cssxref("overflow")}} and {{cssxref("white-space")}}. For example:
```
Comment (teoli2003) [inline files/en-us/web/css/text-overflow/index.md:19]:
```suggestion
The `text-overflow` property doesn't force an overflow to occur. To make text overflow its container, you have to set other CSS properties: {{cssxref("overflow")}} and {{cssxref("white-space")}}. For example:
```

**Comment by teoli2003** [comment]
I think this is better, what do you think?

**Comment by idorenyinudoh** [comment]
looks good 👍🏽

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

**Pull request merged**